src/Style/ItemStyleTrait.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace PhpSchool\CliMenu\Style;
+
+trait ItemStyleTrait
+{
+    protected $markerOn = '';
+
+    protected $markerOff = '';
+
+    protected $itemExtra = '';
+
+    protected $displaysExtra = false;
+
+    protected $custom = false;
+
+    public function getIsCustom() : bool
```

Comment:
When moving this back to the item can we keep it like it is in `MenuStyle` with `hasChangedFromDefaults` ?